api/app/tags/api.py

```diff
@@ -109,6 +109,22 @@ def put(self, event_id):
         tag_repository.commit()
 
         return _serialize_tag_detail(tag), 200
+
+    @staticmethod
```

Comment:
I'd put these two methods on the Tag model itself, since it's behaviour that "belongs" to a Tag entity rather than an API.